website/src/pages/StatusPage/StatusPage.tsx

```diff
@@ -84,42 +99,64 @@ export default function StatusPage() {
       <div className="border-accent border mt-5" />
 
       {/* FILTERS OPTIONS*/}
-      <div>
-        <label>
-          Type:
-          <select name="type" value={filters.type} onChange={handleFilterChange} className="bg-accent rounded-xl p-1">
-            <option value="">All</option>
-            {[...new Set(dataPreviousExe.map((item: any) => item.type_of))].map((type) => (
-                <option key={type} value={type}>
+
+      <div className="flex p-5 gap-4 lg:pl-20">
+        <Select
+          value={filters.type}
+          onValueChange={(value) => handleFilterChange("type", value)}
+        >
+          <SelectTrigger className="w-[180px]">
+            <SelectValue placeholder="Type" />
+          </SelectTrigger>
+          <SelectContent>
+            <SelectItem value="All">All</SelectItem>
+            {[...new Set(datas.map((item: any) => item.type_of))].map(
```

Comment:
Same thing applies to the occurrences of `datas`

Reply:
resolved in 
2a64125